.github/workflows/backend-coverage.yaml

```
@@ -74,9 +74,9 @@ jobs:
         working-directory: ${{ env.backend-directory }}
         env:
           DATABASE_URL: "postgres://postgres:postgres@localhost:${{ job.services.postgres.ports[5432] }}/postgres"
```

actionlint (1.7.7)

property access of object must be type of string but got "number"

(expression)